packages/cli/create-strapi-app/src/utils/usage.ts

```diff
@@ -15,17 +16,19 @@ const getProperties = (scope: Scope, error?: TrackError) => {
   const eventProperties = {
     error: typeof error === 'string' ? error : error && error.message,
   };
+
   const userProperties = {
     os: os.type(),
     osPlatform: os.platform(),
     osArch: os.arch(),
     osRelease: os.release(),
     nodeVersion: process.versions.node,
   };
+
   const groupProperties = {
     version: scope.strapiVersion,
     docker: scope.docker,
-    // useYarn: scope.useYarn,
+    useYarn: scope.packageManager === 'yarn',
```

Comment:
Should we also add `usePnpm` and `useNpm` since they're supported elsewhere?

Reply:
i'll ask the product team if they still need the property

Reply:
I added the pkgManager & kept the old useYarn to avoid breaking charts